products/_bundle/templates/puppet/README.md.erb

```diff
@@ -49,6 +49,21 @@ Puppet module installer:
 
     puppet module install google/cloud
 
+Since these modules depend on the `googleauth` and `google-api-client` gems,
+you will also need to install those, with
+
+    /opt/puppetlabs/puppet/bin/gem install googleauth google-api-client
```

Comment:
is `gem` always going to live in that directory?

Reply:
Yep!